src/main/java/uk/gov/justice/digital/delius/data/api/Custody.java

```diff
@@ -20,4 +22,6 @@
     private CustodyRelatedKeyDates keyDates;
     @ApiModelProperty(value = "Custodial status")
     private KeyValue status;
+    @ApiModelProperty(value = "Date when related sentence started")
```

Comment:
added so custody objects can be distinguished when two sentences (events) combine to single time in prison